dashboard/src/proxy.html

```diff
@@ -111,6 +112,15 @@
             </div>
         </div>
     </div>
+
+    <Section id="proxyHomeSection" class="app-tab"><h1>Home</h1></Section>
```

Comment:
Since `proxyHomeSection` and other IDs are now part of individual typescript files, how about we dynamically add `<Section/>` on page load.  This way we'll be able to remove hardcoded IDs from html.